### PR TITLE
Created gcd.cpp

### PR DESCRIPTION
**Pull request by AdharshM**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- C++14

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by somya-kapoor** [comment]
Please mention the issue number you are trying to fix.
Also, complete the PR checklist properly.

File: Gcd/gcd.cpp [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+int main()
+{
+  ios_base::sync_with_stdio(0);
+  cin.tie(0);
+  cout.tie(0);
+  int a,b;
+  cin>>a>>b;
+  cout<<__gcd(a,b)<<"\n";
+  return 0;
+}
```
Comment (jainaman224) [inline Gcd/gcd.cpp]: Sample input out are missing.

File: Gcd/gcd.cpp [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#include <bits/stdc++.h>
+using namespace std;
+int main()
+{
+  ios_base::sync_with_stdio(0);
```
Comment (jainaman224) [inline Gcd/gcd.cpp]: Please remove this. Use C++ standard cin and cout. This is confusing for learning purposes.

File: Gcd/gcd.cpp [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  cin.tie(0);
+  cout.tie(0);
+  int a,b;
+  cin>>a>>b;
+  cout<<__gcd(a,b)<<"\n";
```
Comment (jainaman224) [inline Gcd/gcd.cpp]: Write gcd function instead using default one.

File: Gcd/gcd.cpp [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#include <bits/stdc++.h>
```
Comment (jainaman224) [inline Gcd/gcd.cpp]: Fix file and folder name.

File: Gcd/gcd.cpp [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+using namespace std;
```
Comment (jainaman224) [inline Gcd/gcd.cpp]: I think this is already implemented. Please check.

**Comment by jainaman224** [comment]
@AdharshM Are you working on it?

**Comment by jainaman224** [comment]
@AdharshM We have to close issue, if you remain unresponsive.

**Comment by AdharshM** [comment]
Dear Aman Jain,

Sorry for being unresponsive for the mails, you have sent. I wasn't able to
respond due to some personal reasons.

Regarding the code, which I made for your repository, it seems that there
are some issues with the code that has to be resolved. Currently, due to
some academic reasons, I am unable to contribute to the repository.
Therefore, you may close the issue of gcd.cpp.

Regards,
Adharsh M

**Comment by AdharshM** [comment]
Once again, Apologies from my side, for being unresponsive for a long while.

**Pull request closed without merging**